integration-tests/utils/Applications.ts

```diff
@@ -40,17 +43,39 @@ export class Applications {
     cy.testA11y(`Select source form`);
   }
 
-  static createComponent(publicGitRepo: string, componentName: string) {
+  static createComponent(publicGitRepo: string, componentName: string, sendPR: boolean = false) {
     addComponentStep(publicGitRepo);
-    reviewComponentsStep(componentName);
+    reviewComponentsStep(componentName, sendPR);
   }
 
+
   static createdComponentExists(componentName: string, applicationName: string) {
     this.goToComponentsTab();
 
     Common.verifyPageTitle(applicationName);
     Common.waitForLoad();
     this.getComponentListItem(componentName).should('exist');
+
+    // if (strictChecking) {
+    //   cy.get(componentsTabPO.componentListItem.replace('{0}', componentName)).contains(/.*Build Running.*/, { timeout: 5000 });
+    // }
```

Comment:
remove or rework, depending if it's needed or not

Reply:
removed, and created a new function `checkComponentStatus()` with similar functionality